torch/_dynamo/convert_frame.py

```diff
@@ -1120,7 +1101,6 @@ def clean_for_json(d: Dict[str, Any]) -> Dict[str, Any]:
 
             config_dict = clean_for_json(config.get_config_copy())
             metrics = {
-                "compile_id": str(compile_id),
```

Comment:
All these fields are common to the logging for the backwards, so they're all moving to `record_compilation_metrics`